tests/test_database_actions.py

```
@@ -1,217 +1,332 @@
+        # Check parameters if provided
+        if expected_params:
+            params = call_args[1].get("params", {})
+            for key, value in expected_params.items():
+                self.assertEqual(params[key], value)
+
+    @mock.patch("pandas.read_sql")
+    def test_select_variants_from_db_with_sql_check(self, pd_read_sql_mock):
+        mock_engine = mock.MagicMock()
+        pd_read_sql_mock.return_value = self.df
+
+        return_df = db.select_variants_from_db("1234", mock_engine, "NOT NULL")
```

Ruff (0.14.7)

Local variable `return_df` is assigned to but never used

Remove assignment to unused variable `return_df`

(F841)